compile error getarch_2nd.c w AMD 2950X

Hi,
I have the following error with AMD Ryzen Threadripper 2950X 16-Core Processor.
First, I used TARGET=ZEN, but reading [issue 1664](https://github.com/xianyi/OpenBLAS/issues/1664) I also tried TARGET=HASWELL.
What to do?
Thanks, Tamas

make -j16 TARGET=HASWELL
...
sudo make PREFIX=/opt/OpenBLAS-0.2.20 install


getarch_2nd.c: In function ‘main’:
getarch_2nd.c:12:35: error: ‘SGEMM_DEFAULT_UNROLL_M’ undeclared (first use in this function)
     printf("SGEMM_UNROLL_M=%d\n", SGEMM_DEFAULT_UNROLL_M);
...
getarch_2nd.c:69:50: error: ‘SGEMM_DEFAULT_Q’ undeclared (first use in this function)
     printf("#define SLOCAL_BUFFER_SIZE\t%ld\n", (SGEMM_DEFAULT_Q * SGEMM_DEFAULT_UNROLL_N * 4 * 1 *  sizeof(float)));
...

make: *** [getarch_2nd] Error 1
make -j 32 -f Makefile.install install
make[1]: Entering directory '/home/hegedus/src/OpenBLAS-0.2.20'
Generating openblas_config.h in /opt/OpenBLAS-0.2.20/include
Generating f77blas.h in /opt/OpenBLAS-0.2.20/include
Generating cblas.h in /opt/OpenBLAS-0.2.20/include
Copying the static library to /opt/OpenBLAS-0.2.20/lib
Copying the shared library to /opt/OpenBLAS-0.2.20/lib
Generating openblas.pc in /opt/OpenBLAS-0.2.20/lib/pkgconfig
Generating OpenBLASConfig.cmake in /opt/OpenBLAS-0.2.20/lib/cmake/openblas
Generating OpenBLASConfigVersion.cmake in /opt/OpenBLAS-0.2.20/lib/cmake/openblas
Install OK!

--- a/cpuid_x86.c
+++ b/cpuid_x86.c
@@ -2009,6 +2009,8 @@ int get_coretype(void){
 	switch (model) {
 	case 1:
 	  // AMD Ryzen
+	case 8:
+	  // Ryzen 2		
 	  if(support_avx())
 #ifndef NO_AVX2
 	    return CORE_ZEN;

